django/core/static/css/custom_large.css

```diff
@@ -0,0 +1,17 @@
+@media (min-width: 992px) {
+    /* ---------- Navigation ---------- */
```

Comment:
Do these comments help? I'd suggest just deleting lines 2-11. Also in django/core/static/css/custom_small.css

Reply:
They're there ready for when I make the site responsive. I'll delete them but they'll just get added back